pkg/constants/constants.go

```diff
@@ -107,11 +106,11 @@ func GetXorrisoBooloaderArgs(root string) []string {
 		"-boot_image", "any", "platform_id=0x00",
 		"-boot_image", "any", "emul_type=no_emulation",
 		"-boot_image", "any", "load_size=2048",
-		"-append_partition", "2", "0xef", filepath.Join(root, IsoEFIPath),
 		"-boot_image", "any", "next",
-		"-boot_image", "any", "efi_path=--interval:appended_partition_2:all::",
 		"-boot_image", "any", "platform_id=0xef",
 		"-boot_image", "any", "emul_type=no_emulation",
+		"-boot_image", "any", "efi_path=" + IsoEFIPath, // Tell where the efi path is
```

Comment:
this is just an improvement over what we had.
Before:
 - uefi.img in the disc
 - second partition in the disc with uefi contents

Now:
 - just the uefi.img in the disc